components/NuxtLogo.vue

```diff
@@ -14,6 +14,6 @@
 
 <style>
 .nuxt-logo {
-  height: 180px;
+  height: 20px;
```

Comment:
Avoid pixels, use `rem`s (and why not Tailwind?)

Reply:
This was default code on project setup.